README.md

```diff
@@ -133,51 +133,58 @@ We will also release wrappers to some useful Java libraries frequently used in t
+-   Flexibility, as the user can switch between different data structures;
+-   Operations for data preprocessing, pattern mining and data visualization.
 
 ---
 
 ## Creating a Virtual Environment
 
 It is recommended to create a virtual environment to use pymove. Requirements: Anaconda Python distribution installed and accessible
 
-1. In the terminal client enter the following where `yourenvname` is the name you want to call your environment, and replace `x.x` with the Python version you wish to use. (To see a list of available python versions first, type conda search "^python$" and press enter.)
-    - `conda create -n <yourenvname> python=x.x`
-    - Press y to proceed. This will install the Python version and all the associated anaconda packaged libraries at `path_to_your_anaconda_location/anaconda/envs/yourenvname`
-2. Activate your virtual environment. To activate or switch into your virtual environment, simply type the following where yourenvname is the name you gave to your environment at creation.
-    - `conda activate <yourenvname>`
-3. Now install the package from either `conda`, `pip` or `github`
+1.  In the terminal client enter the following where `yourenvname` is the name you want to call your environment, and replace `x.x` with the Python version you wish to use. (To see a list of available python versions first, type conda search "^python$" and press enter.)
+    -   `conda create -n <yourenvname> python=x.x`
+    -   Press y to proceed. This will install the Python version and all the associated anaconda packaged libraries at `path_to_your_anaconda_location/anaconda/envs/yourenvname`
+
+1.  Activate your virtual environment. To activate or switch into your virtual environment, simply type the following where yourenvname is the name you gave to your environment at creation.
+    -   `conda activate <yourenvname>`
+
+1.  Now install the package from either `conda`, `pip` or `github`
 
 ---
 
 ## [Conda](https://anaconda.org/conda-forge/pymove) instalation
 
-1. `conda install -c conda-forge pymove`
+1.  `conda install -c conda-forge pymove`
 
 ## [Pip](https://pypi.org/project/pymove) installation
 
-1. `pip install pymove`
+1.  `pip install pymove`
 
 ---
 
 ## [Github](https://github.com/InsightLab/PyMove) installation
 
-1. Clone this repository
-    - `git clone https://github.com/InsightLab/PyMove`
-2. Make a branch developer
-    - `git branch developer`
-3. Switch to a new branch
-    - `git checkout developer`
-4. Make a pull of branch
-    - `git pull origin developer`
-5. Switch to folder PyMove
-    - `cd PyMove`
-6. Install pymove in developer mode
-    - `pip install -e .`
+1.  Clone this repository
+    -   `git clone https://github.com/InsightLab/PyMove`
+
+1.  Make a branch developer
+    -   `git branch developer`
+
+1.  Switch to a new branch
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[ordered-list-marker-value] Marker should be `3`, was `1`](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5537263)